packages/fire_capture/PKGBUILD

```
@@ -1,29 +1,58 @@
-# Maintainer: Mattia Procopio <matto.astro@gmail.com>
+# Maintainer: Ed Lee <ed@thefamilee.co.uk>
+# Previous maintainer: Mattia Procopio <matto.astro@gmail.com>
 # Original AUR PKGBUILD mantainer Daniel Reibl <janos.daniel.reibl@pm.me>
+# Most changes from previous work thanks to Stéphane Carlin
 # This build contains some changes that allow FireCapture to work on aarch64 and start with no path issues
 pkgname=firecapture
-pkgver=2.7.14
+pkgver=2.7.14a
 pkgrel=1
 pkgdesc="The leading planetary capture tool"
 arch=(aarch64)
-url="http://www.firecapture.de"
+url="https://www.firecapture.de"
 license=('custom:firecapture')
-depends=('libusb')
-source=(fire_capture.deb::https://www.thefamilee.co.uk/public/Astro/firecapture_${pkgver}_aarch64.deb)
-sha256sums=(3bbcabe388c238094e81580a7d93108248b01687c76a4dfdb98b7518ed3fef51)
-_filename=fire_capture.deb
+depends=('libusb' 'jdk-openjdk' 'indi-3rdparty-libs')
+source=("${pkgname}_${pkgver}_${arch}.deb::https://github.com/devDucks/astroarch-pkgs/raw/refs/heads/main/packages/fire_capture/${pkgname}_${pkgver}_${arch}.deb")
```

I don't think you need to store a separate copy of the .deb in this repository. If you just change this line to 
```suggestion
source=("${pkgname}_${pkgver}_${arch}.deb::https://www.thefamilee.co.uk/public/${pkgname}_${pkgver}_${arch}.deb")
```
it should work to download it directly from the source, since https://www.thefamilee.co.uk/public/firecapture_2.7.14a_aarch64.deb resolves correctly. In general, this is preferable to storing an independent copy within this repo that needs to be replaced every time a new upstream version is released.

---

Actually no, sorry. The package at thefamilee.co.uk was not official source, only a temporary package left over from debugging. The official source is only available from firecapture.de but it's behind a challenge/response to prevent automatic downloads. After discussing with Mattia it was agreed to host a copy.